dashboard/config/locales/en.yml

```diff
@@ -1481,3 +1481,11 @@ en:
     alt_text_with_name: "Certificate for %{student_name} for completion of %{course_name}"
     alt_text_no_name: "Certificate for completion of %{course_name}"
     one_hour_of_code: "One Hour of Code"
+  child_account:
+    parental_permission_request:
+      errors:
+        invalid_parent_email: The parent/guardian's email address is invalid
+        already_granted: Parental permission has already been granted
+        parent_email_is_own: A request cannot be sent to your own email address
+        resend_limit_reached: The limit for resend requests has been reached
+        daily_limit_reached: The daily request limit has been reached
```

Comment:
Could we have all these new strings ordered alphabetically, including the parent keys? It would make them more discoverable. I have the feeling that we already have some strings in this file for CPA, but it is such a massive file is hard to know what already exists.